### DISCUSSION TITLE
Welcome to KVKCalendar Discussions!

### DISCUSSION THREAD

**Discussion by kvyatkovskys** [kvyatkovskys/KVKCalendar#113] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ddferrando** [comment]
Hi.
First of all thanks this great job.
I am wondering if there is a way to create and move event snapint them to specific round time.
Let's say half an hour for example.

Thanks DAvid

**Reply by kvyatkovskys** [reply]
> Hi.
> First of all thanks this great job.
> I am wondering if there is a way to create and move event snapint them to specific round time.
> Let's say half an hour for example.
> 
> Thanks DAvid

Hello 🙂
Sorry for a long answer. 

Thanks a lot 🙂. could you give a more details, pls? 
BTW, you can create an event for specific time and then to move the event as you want
